code/controllers/subsystem/verb_manager.dm

```diff
@@ -0,0 +1,86 @@
+ * 3. control is passed to byond after all of our code's procs stop execution for this tick
+ * 4. a few small things happen in byond internals
+ * 5. SendMaps is called for this tick, which processes the game state for all clients connected to the game and handles sending them changes
+ * 	in appearances within their view range. This is expensive and takes up a significant portion of our tick, about 0.45% per connected player
+ * 	as of 3/20/2022. meaning that with 50 players, 22.5% of our tick is being used up by just SendMaps, after all of our code has stopped executing.
+ *	Thats only the average across all rounds, for most highpop rounds it can look like 0.6% of the tick per player, which is 30% for 50 players.
+ * 6. After SendMaps ends, client verbs sent to the server are executed, and its the last major step before the next tick begins.
+ *	During the course of the tick, a client can send a command to the server saying that they have executed any verb. The actual code defined
+ *	for that /verb/name() proc isnt executed until this point, and the way the MC is designed makes this especially likely to make verbs
+ *	"overrun" the bounds of the tick they executed in, stopping the other tick from starting and thus delaying the MC firing in that tick.
+ *
+ * The way the MC allots its time is via TICK_LIMIT_RUNNING, it simply subtracts the cost of SendMaps (MAPTICK_LAST_INTERNAL_TICK_USAGE)
+ * plus TICK_BYOND_RESERVE from the tick and uses up to that amount of time (minus the percentage of the tick used by the time it executes subsystems)
+ * on subsystems running cool things like atmospherics or Life or SSInput or whatever.
+ *
+ * Without this subsystem, verbs are likely to cause overtime if the MC uses all of the time it has alloted for itself in the tick, and SendMaps
+ * uses as much as its expected to, and an expensive verb ends up executing that tick. This is because the MC is completely blind to the cost of
+ * verbs, it can't account for it at all. The only chance for verbs to not cause overtime in a tick where the MC used as much of the tick
+ * as it alloted itself and where SendMaps costed as much as it was expected to is if the verb(s) take less than TICK_BYOND_RESERVE percent of
+ * the tick, which isnt much. Not to mention if SendMaps takes more than 30% of the tick and the MC forces itself to take at least 70% of the
+ * normal tick duration which causes ticks to naturally overrun even in the absence of verbs.
+ *
+ * With this subsystem, the MC can account for the cost of verbs and thus stop major overruns of ticks. This means that the most important subsystems
+ * like SSinput can start at the same time they were supposed to, leading to a smoother experience for the player since ticks arent riddled with
+ * minor hangs over and over again.
+ */
+SUBSYSTEM_DEF(verb_manager)
+	name = "Verb Manager"
+	wait = 1
+	flags = SS_TICKER
+	priority = FIRE_PRIORITY_DELAYED_VERBS
+	runlevels = ALL//trollface
```

Comment:
```suggestion
	runlevels = ALL //trollface
```
help my brain is broken

Reply:
so is mine, now it uses RUNLEVEL_DEFAULT | RUNLEVEL_INIT